lib/matplotlib/tests/test_text.py

```diff
@@ -421,3 +421,11 @@ def test_font_scaling():
 
     for i, fs in enumerate(range(4, 43, 2)):
         ax.text(0.1, i*30, "{fs} pt font size".format(fs=fs), fontsize=fs)
+
+
+@pytest.mark.parametrize('spacing1, spacing2', [(2, 0.4), (0.4, 2)])
```

Comment:
The bug with the text line spacing was that adding multiple text with different line spacing would not be correctly rendered. So this test would check that the intended line spacing for each the texts was applied. The second set of parameters was to check that calling text() in a different order would still produce the same result.